labgrid/remote/client.py

```diff
@@ -68,11 +68,17 @@ def onConnect(self):
         self.monitor = self.config.extra.get('monitor', False)
         enable_tcp_nodelay(self)
         self.join(
-            self.config.realm, authmethods=["ticket"],
-            authid=f"client/{self.gethostname()}/{self.getuser()}"
+            self.config.realm,
+            authmethods=["anonymous", "ticket"],
+            authid=f"client/{self.gethostname()}/{self.getuser()}",
+            authextra={"authid": f"client/{self.gethostname()}/{self.getuser()}"},
         )
 
     def onChallenge(self, challenge):
+        import warnings
+        warnings.warn("Ticket authentication is deprecated. Please update your coordinator.",
+                      DeprecationWarning)
+        logging.warning("Ticket authentication is deprecated. Please update your coordinator.")
```

Comment:
I am not sure if this is too verbose. On the other hand, users might miss the deprecation if we don't warn loud enough.

Reply:
Yes, Warning load and clear is the best move here.